### ISSUE TITLE
How to turn off the LCD at night?

### ISSUE THREAD

**Issue by tggo** [tjaworski/AceMagic-S1-LED-TFT-Linux#24]
Is there a way to turn off the LSD backlight so that it doesn't glow at night?

**Comment by tjaworski** [comment]
not that I found. best you can do is put a blank screen with a black background. but even that is still bright.

**Comment by shadowpho** [comment]
I wonder if the MCU has a built-in command for turn-off...

**Comment by tjaworski** [comment]
The Holtek device is powered through the internal USB connection, which requires the USB port to be powered off for the device to fully shut down. When the external power is disconnected from the unit and then reconnected, the LCD remains off until the power button is pressed. I suspect that the LCD backlight is directly connected to the USB power, which would explain why the firmware cannot control the backlight independently.

**Comment by tjaworski** [comment]
if the guys in china Shenzhen CYX Industrial Co are reading this, and would like to chime in, I'm all ears :)

**Comment by renzo-gh** [comment]
On the TFT LCD module wiring diagram there is an EN_BL pin connected to the C.I. Led Driver HT7936 , so it should be possible to turn off the backlight if you can change the state, Low should be turned off

**Comment by renzo-gh** [comment]
https://www.holtek.com/WebAPI/187541/ESK32-A2A31_UserManualv100.pdf   link of manual

**Comment by tjaworski** [comment]
right, but that pin would be controlled by the firmware on the board, and unless they expose the endpoint to allow to toggle that io pin, we don't have access to it.

**Comment by renzo-gh** [comment]
sure, thanks for the prompt reply, I just wanted to point out the possibility...
I'm more hardware oriented, if possible managing with firmware could be a way

**Comment by tjaworski** [comment]
right, but unless someone can dump the firmware and we can disassemble it to see how it works, the functionality it has is what we have right now. best case scenario they already have a command to turn off the power to the lcd. worse scenario, we need to make our own firmware and find a way to flash it. either way, getting the firmware would be step one. either by brute force or some nicer way. maybe someone reading this is more familiar with this.
